media/css/readthedocs-doc-embed.css

```diff
@@ -7,6 +7,9 @@
     display: block;
 
     bottom: 50px;
+
+    /* Workaround for mkdocs which set a specific height for this element */
+    height: auto;
```

Comment:
This fixes the bug seen here which is currently visible on https://mkdocs.readthedocs.io/en/stable/

<img width="906" alt="screen shot 2018-08-22 at 10 16 18 am" src="https://user-images.githubusercontent.com/185043/44479118-806c8900-a5f4-11e8-957d-4522ccbb074e.png">